app/src/main/res/raw/menu.json

```diff
@@ -387,6 +387,94 @@
 				],
 				"icon": "https://www.opensouthcode.org/logo.png"
 			}
+		},
+		{
+			"version": 2017230500,
+			"url": "https://raw.githubusercontent.com/heysadboy/Giggity-Parser/master/Parser/sample.xml",
+			"title": "FOSSASIA 2016",
+			"start": "2016-08-10",
+			"end": "2016-08-12",
+			"metadata": {
+				"links": [
+					{
+						"url": "http://fossasia.org",
+						"title": "Website"
+					}
+				],
+              "icon": "https://raw.githubusercontent.com/fossasia/open-event-android/development/android/app/src/main/res/drawable/splash_logo.png",
+              "rooms": [
+
+					{
+						"name": "Dalton Hall",
+						"latlon": [1.333169, 103.735516]
+
+					},
+					{
+						"name": "Lewis Lab",
+						"latlon": [1.332835, 103.735677]
+					},
+					{
+						"name": "Digital Design Studio",
+						"latlon": [1.332668, 103.735725]
+					},
+					{
+						"name": "Digital Design Studio",
+						"latlon": [1.332835, 103.735677]
+					},
+					{
+						"name": "Observatory Room",
+						"latlon": [1.332913, 103.735716]
+					},
+					{
+						"name": "Eco Garden Lab",
+						"latlon": [null, null]
```

Comment:
No need to include these BTW, and I haven't tested how Giggity would behave on this. I'll remove it before merging.